controllers/helpers.go

```diff
@@ -29,7 +29,7 @@ func (r *EtcdadmConfigReconciler) MachineToBootstrapMapFunc(o client.Object) []c
 
 // ClusterToEtcdadmConfigs is a handler.ToRequestsFunc to be used to enqeue
 // requests for reconciliation of EtcdadmConfigs.
-func (r *EtcdadmConfigReconciler) ClusterToEtcdadmConfigs(o client.Object) []ctrl.Request {
+func (r *EtcdadmConfigReconciler) ClusterToEtcdadmConfigs(ctx context.Context, o client.Object) []ctrl.Request {
```

Comment:
you can now use this context in the body of the methods instead of context.Background :)

Reply:
oh nice catch!